functionaltests/8_15_test.go

```diff
@@ -60,11 +60,16 @@ func TestUpgrade_8_15_4_to_8_16_0(t *testing.T) {
 	})
 
 	var escfg esclient.Config
-	tf.Output("apm_url", &escfg.APMServerURL)
-	tf.Output("es_url", &escfg.ElasticsearchURL)
-	tf.Output("username", &escfg.Username)
-	tf.Output("password", &escfg.Password)
-	tf.Output("kb_url", &escfg.KibanaURL)
+	err = tf.Output("apm_url", &escfg.APMServerURL)
+	require.NoError(t, err)
```

Comment:
Minor nit to reduce the number of lines, I think it's still quite readable:
```suggestion
	require.NoError(t, tf.Output("apm_url", &escfg.APMServerURL))
```

Reply:
Agreed and updated